cosmwasm/packages/wasmi-runtime/Makefile

```diff
@@ -118,11 +119,11 @@ $(Enclave_EDL_Products): Enclave.edl
 	sgx_edger8r --trusted $^ --search-path $(SGX_SDK)/include --search-path $(CUSTOM_EDL_PATH) --trusted-dir ./
 
 check:
-	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS="-Z force-unstable-if-unmarked" xargo check --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
+	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS=$(Rust_Flags) xargo check --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
 
 clippy:
-	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS="-Z force-unstable-if-unmarked -D warnings" xargo clippy --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
+	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS=$(Rust_Flags) xargo clippy --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx -- -D warnings
```

Comment:
Did you check this works? I tried it and it didn't actually break on warnings

Reply:
Weird, we tested it and it worked cc @reuvenpo

Reply:
Yeah just checked and it seems to work. Idk, maybe I did something wrong last time I tried